<?xml version="1.0"?>
<launch>
    <!-- EKF Ackermann Odometry Node -->
    <node name="ekf_ackermann_odom" pkg="your_package_name" type="ekf_ackermann_odom_node" output="screen">
        <!-- 설정 파일 로드 -->
        <rosparam file="$(find your_package_name)/config/ekf_config.yaml" command="load" />
        
        <!-- 개별 파라미터 오버라이드 (필요시) -->
        <!-- <param name="wheel_base" value="0.26" /> -->
        <!-- <param name="use_tf" value="true" /> -->
        
        <!-- 토픽명 오버라이드 (필요시) -->
        <!-- <param name="odom_topic" value="/robot/odom" /> -->
        <!-- <param name="drive_topic" value="/robot/ackermann_cmd" /> -->
        <!-- <param name="imu_topic" value="/robot/imu/data" /> -->
        <!-- <param name="odom_frame" value="robot_odom" /> -->
        <!-- <param name="base_frame" value="robot_base_link" /> -->
    </node>

    <!-- 시각화를 위한 RViz (선택사항) -->
    <!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find your_package_name)/rviz/ekf_odom.rviz" /> -->
</launch>